lab-jaren/server.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const http = require('http');
+const url = require('url');
+const querystring = require('querystring');
+const cowsay = require('cowsay');
+
+const parseBodyHandler = require('./lib/parse-body-handler.js');
+const badRequestHandler = require('./lib/bad-request-handler.js');
+const cowsayHTML = require('./lib/cowsay-html.js');
+
+const PORT = process.env.PORT || 3000;
+const server = http.createServer(function(message, response) {
+  /* parse url and query */
```

Comment:
Comments are great! Try to get in the process of setting up another branch that you use to have your comments and submitting a clean PR for my review. 

I just want people to establish good habits of doing that before they do something "real world" related like deploying code live.